lib/src/form_builder_validators.dart

```diff
@@ -172,6 +172,48 @@ class FormBuilderValidators {
     };
   }
 
+  /// [FormFieldValidator] that requires the words count of the field's value to be
+  /// greater than or equal to the provided minimum count.
+  static FormFieldValidator<String> minWordsCount(
+    int minCount, {
+    bool allowEmpty = false,
+    String? errorText,
+  }) {
+    assert(minCount > 0);
+    return (valueCandidate) {
+      int valueWordsCount = 0;
+
+      if (valueCandidate != null) {
+        if (valueCandidate.trim().isEmpty) {
+          valueWordsCount = 0;
+        } else {
+          valueWordsCount = valueCandidate.trim().split(' ').length;
+        }
+      }
+
+      return valueWordsCount < minCount && (!allowEmpty || valueWordsCount > 0)
+          ? errorText ??
+              FormBuilderLocalizations.current.minWordsCountErrorText(minCount)
+          : null;
+    };
+  }
+
+  /// [FormFieldValidator] that requires the words count of the field's value to be
+  /// less than or equal to the provided maximum count.
+  static FormFieldValidator<String> maxWordsCount(
+    int maxCount, {
+    String? errorText,
+  }) {
+    assert(maxCount > 0);
```

Comment:
Please add a assert detail message to explain better why got the assert and how solve

Reply:
Added ✅